CODE_OF_CONDUCT.md

```diff
@@ -15,6 +15,14 @@ A supplemental goal of this Code of Conduct is to increase open [source/culture/
 
 Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
 
+To provide equal access of information to all, the Mobile Development Club shall refrain from actions and methods of broadcasts that unnecessarily exclude certain demographics or individuals from accessing information released and announced by the RSO
+(examples against this clause include but are not limited to making announcements solely to major and department-specific communication mailing-lists and other communication groups OR to explicitly state or
+conduct outreach in a fashion that suggests that the RSO is "exclusive to people in computing").
+
+To ensure that all are welcomed at the Mobile Development Club, the RSO shall maintain an open, free, and neutral stance on the freedom of expression. RSO meetings and gatherings are expected to follow the rough topics of discussion
+as outlined by the agenda and the material presented. However, no **forceful** attempts shall be made to restrict the will to express of any individual or entity present. Non-forceful reactions to any discharge of expression, including but not limited
+to frowning at the expresser and expressing general disapproval are not entirely constructive to a welcoming atmosphere, but nonetheless shall also **NOT** be forcefully restricted to uphold the freedom of expression of all.
```

Comment:
These 2 paragraphs mainly concerns how the Board/mentors should behave rather than all individual in the community. Though we should follow the requirements outlined in these 2 paragraphs and the CCoC does set guidelines for how leadership should act, the main purpose is to guide how community at large should expect, not just the leadership. 

Could you please justify why these 2 paragraphs works better in CCoC instead of a separate doc regulating how leadership/those representing "the RSO" should carry out the duties? @Lucas-ZX-W